docs/concepts/actors.mdx

```diff
@@ -24,47 +19,41 @@ The academic definition of actors calls for them to be able to perform the follo
 * Make local decisions
 * Choose how to react to the next message
 
-Implicit in this list is the rule that actors can't modify or directly read the state of other actors. The only way data is exchanged between entities in this system is through messages. There are a _ton_ of amazing resources on this concept, and if you're interested we highly recommend that you track down some of the references in the Wikipedia article.
+Implicit in this list is the rule that actors can't modify or directly read the state of other actors. The only way data is exchanged between entities in this system is through messages. There are a _ton_ of amazing resources on this concept, and if you're interested we highly recommend that you track down some of the references in the [Wikipedia article](https://en.wikipedia.org/wiki/Actor_model).
 
-**Note:** While it's possible to implement the actor model using these strict, academic
-constraints, wasmCloud must temper them with some practical concerns that arise
-from the need to manage **secure** and **resilient** distributed workloads.
+**Note:** While it's possible to implement the actor model using these strict, academic constraints, wasmCloud must temper them with some practical concerns that arise from the need to manage **secure** and **resilient** distributed workloads.
 
-## Stateless Actors
-wasmCloud actors are **stateless**, allowing hundreds or thousands of copies
-of an actor to be horizontally scaled across a cluster to meet compute demands.
-When an actor needs state, it obtains it easily through the use of a capability provider.
-By using capability providers, wasmCloud can ensure that no matter
-which instance of an actor is invoked, or where it's running in the cluster,
-it can still dynamically respond to the state of the larger application.
+### Stateless By Design
+
+wasmCloud actors are **stateless**, allowing hundreds or thousands of copies of an actor to be horizontally scaled across a cluster to meet compute demands. When an actor needs state, it obtains it easily through the use of a capability provider. By using capability providers, wasmCloud can ensure that no matter which instance of an actor is invoked, or where it's running in the cluster, it can still dynamically respond to the state of the larger application.
```

Comment:
Link to capability providers doc here?

Also some spacing between the first and second sentence might be good?